exercises/more_string_manipulation/challenge_080.py

```diff
@@ -0,0 +1,40 @@
+# MORE STRING MANIPULATION
+
+# Challenge 080
+"""
+Ask the user to enter their first name and then display the length of their first name.
+Then ask for their surname and display the length of their username. Join their first name
+and surname together with a space between and display the result.
+Finally, display the length of their full name (including the space).
+"""
+
+
+def processing(first_name, last_name):
+    full_name = first_name + " " + last_name
+    length_first_name = len(first_name)
+    length_surname = len(last_name)
+    length_full_name = len(full_name)
+    return length_first_name, length_surname, length_full_name, full_name
+
+
+def display_length_name():
+    # INPUT
+    user_first_name = input("Please, enter your name: ")
+    user_last_name = input("Enter your last name: ")
+
+    # PROCESSING
+    (
+        length_user_first_name,
+        length_user_last_name,
+        length_full_name,
+        full_name,
+    ) = processing(user_first_name, user_last_name)
+
+    # OUTPUT
+    print(f"Your name has {length_user_first_name} characters in it.")
```

Comment:
The challenge asks you to display the length after the input and before the next input. 

I think it wants you to abstract the collection and reflection of the information rather than what you've done here which is rather literally separated the input collection and the response

Reply:
I have refactored the code